FRW/Services/Network/FlowNetwork.swift

```diff
@@ -1164,6 +1164,13 @@ extension FlowNetwork {
         ).decode(Flow.StorageInfo.self)
         return response
     }
+    
+    static func checkAccountInfo() async throws -> Flow.AccountInfo {
+        let address = Flow.Address(hex: WalletManager.shared.getPrimaryWalletAddress() ?? "")
+        let cadence = CadenceManager.shared.current.basic?.getAccountInfo?.toFunc() ?? ""
```

Comment:
let's throw the error if the address or cadence is empty

Reply:
Ok. It doesn't seem to exist an error for invalid cadence, so I've added `invalidCadence` to `LLError`. But let me know if there's another error that fits better.